### ISSUE TITLE
Darwin support gives weird error, references a missing variable

### ISSUE THREAD

**Issue by amosshapira** [counsyl/puppet-python#4]
I'm trying to use the new Darwin support to install pip on OSX 10.10.5, Puppet 3.8.2, ruby 2.0.0p481.

I keep getting the errors:

```
Error: Mac OS X PKG DMG's must specify a package source.
Error: /Stage[main]/Python/Package[python-pip]/ensure: change from absent to present failed: Mac OS X PKG DMG's must specify a package source.
```

To replicate this, I executed as `root`:

```
puppet apply --modulepath=~/modules --debug -e 'include python'
```

Where ~/modules contained the 'python' and 'sys' modules.

As far as I can tell, the error comes from the Package resource defined in line 173 (https://github.com/counsyl/puppet-python/blob/master/manifests/init.pp#L173):

```
package { $pip_package:
  ensure   => $pip_ensure,
  provider => $provider,
  source   => $source,
  require  => $setuptools_require,
}
```

$source is empty. This line is reached because `$ez_setup` is `false` but `$pip_package` is defined.

I'm not yet familiar enough with these tools on OSX to understand what's supposed to happen. All the more recent links I found about installing pip on OSX rely on HomeBrew to do that.

**Comment by amosshapira** [comment]
Is there any chance of this issue being addressed? It's been open for over 6 months now.

**Comment by jdavisp3** [comment]
Sorry for the radio silence here. I believe this is addressed in the 0.9.9 release, which I just made. Can you confirm?

**Comment by amosshapira** [comment]
Thanks.

I've verified that the module is now fixed by repeating the same steps only using the current `master` branch.

BTW - current latest version available from Homebrew is 2.7.13

**Comment by jdavisp3** [comment]
Excellent, thank you @amosshapira
